inst/pages/exercises.qmd

```diff
@@ -1703,17 +1706,27 @@ Create an interactive heatmap.
 #| code-summary: "Show solution"
 #| eval: false
 
-# Load necessary libraries
+# Load required libraries
+library(mia)
 library(heatmaply)
 
-# Create example data
-example_data <- matrix(rnorm(200), nrow = 20, ncol = 10)
+# Load the GlobalPatterns dataset from the mia package
+data("GlobalPatterns", package = "mia")
+
+# Here we extract the first 20 taxa and the first 10 samples as an example
+taxa_data <- assay(GlobalPatterns)[1:20, 1:10]
+taxa_names <- rownames(taxa_data)
+sample_names <- colnames(taxa_data)
 
-# Create interactive heatmap
-heatmaply(example_data)
+# Create the interactive heatmap using heatmaply
+heatmaply(taxa_data,
```

Comment:
If heatmaply does not support SE, then 

1) let's consider if this example brings enough added value to be included in OMA (another option is just to mention that it is available but not show an example)

2) if we keep this in OMA, then I would write more directly
`heatmaply(assay(GlobalPatterns)[1:20, 1:10]), ...)`

or even better, aggregate first to higher taxonomic level with agglomerateByRank() so that you can show the entire data instead of a subset.

Reply:
Alternatively, you can check if there is another smaller data set that could be visualized entirely.